src/Traits/TransactionTrait.php

```diff
@@ -180,14 +215,16 @@ trait TransactionTrait
      *
      * @Assert\Type(type="string")
      *
+     * @deprecated Being removed in favour of $state.
```

Comment:
I'm not sure we're actually deprecating status - it is still used internally. We've just exposed a state property that has more detail for the frontend to display whats going on with transactions

Reply:
Let's pick this up after standup. I'm not fussed either way, but let's make sure we're sure what our strategy for exposing state, status etc is, and apply it consistently.

Reply:
Discussion outcome: We will expose the information, and update docs in both SDK & Payments to indicate exactly how the fields relate.